### ISSUE TITLE
glGetUniformLocationChecked ERROR: Failed to get location of uniform 'opacity'. the compositor might not work correctly.

### ISSUE THREAD

**Issue by ndgnuh** [yshui/picom#309]
### Platform
Archlinux, I'm using `--experimental-backends`.

### GPU, drivers, and screen setup
Intel HD Graphics 620, 2 monitors side-by-side, using `xrandr` to config.
Video drivers: (`pacman -Qs xf86-video`)
```
local/xf86-video-amdgpu 19.1.0-1 (xorg-drivers)
local/xf86-video-ati 1:19.1.0-1 (xorg-drivers)
local/xf86-video-dummy 0.3.8-3 (xorg-drivers)
local/xf86-video-fbdev 0.5.0-1 (xorg-drivers)
local/xf86-video-intel-git 1:2.99.917+892+gc6cb1b19-2 (xorg-drivers)
local/xf86-video-nouveau 1.0.16-1 (xorg-drivers)
local/xf86-video-openchrome 0.6.0-3 (xorg-drivers)
local/xf86-video-qxl 0.1.5-7 (xorg-drivers)
local/xf86-video-vesa 2.4.0-2 (xorg-drivers xorg)
local/xf86-video-vmware 13.3.0-1 (xorg-drivers)
local/xf86-video-voodoo 1.2.5-10 (xorg-drivers)
```

Output of `glxinfo -B`: 
```
name of display: :0
display: :0  screen: 0
direct rendering: Yes
Extended renderer info (GLX_MESA_query_renderer):
    Vendor: Intel Open Source Technology Center (0x8086)
    Device: Mesa DRI Intel(R) HD Graphics 620 (Kaby Lake GT2)  (0x5916)
    Version: 19.3.3
    Accelerated: yes
    Video memory: 3072MB
    Unified memory: yes
    Preferred profile: core (0x1)
    Max core profile version: 4.6
    Max compat profile version: 3.0
    Max GLES1 profile version: 1.1
    Max GLES[23] profile version: 3.2
OpenGL vendor string: Intel Open Source Technology Center
OpenGL renderer string: Mesa DRI Intel(R) HD Graphics 620 (Kaby Lake GT2) 
OpenGL core profile version string: 4.6 (Core Profile) Mesa 19.3.3
OpenGL core profile shading language version string: 4.60
OpenGL core profile context flags: (none)
OpenGL core profile profile mask: core profile
OpenGL version string: 3.0 Mesa 19.3.3
OpenGL shading language version string: 1.30
OpenGL context flags: (none)
OpenGL ES profile version string: OpenGL ES 3.2 Mesa 19.3.3
OpenGL ES profile shading language version string: OpenGL ES GLSL ES 3.20
```
### Environment
I'm using Awesomewm only.

### picom version
Version: latest commit
`picom --diagnostics`'s output:
```
**Version:** v7.5

### Extensions:

* Shape: Yes
* XRandR: Yes
* Present: Present

### Misc:

* Use Overlay: Yes
* Config file used: /home/hung/.config/picom.conf

### Drivers (inaccurate):

Intel
```

### Configuration:
```
############
#  shadow  #
############
shadow = true;
shadow-radius = 11;
shadow-offset-x = -7;
shadow-offset-y = -7;
# shadow-opacity = 0.7;
# shadow-red = 0.0;
# shadow-green = 0.0;
# shadow-blue = 0.0;
shadow-exclude = [
    "argb",
    "name = 'Notification'",
    "class_g ?= 'Notify-osd'",
    "class_g = 'Cairo-clock'",
    "_GTK_FRAME_EXTENTS@:c"
];
# shadow-exclude = "n:e:Notification";
# shadow-exclude-reg = "x10+0+0";
# xinerama-shadow-crop = true;
normal = { shadow = false; };
# menu        = { shadow = false; };
dropdown_menu = { shadow = false; };
popup_menu    = { shadow = false; };
utility       = { shadow = false; };

#############
#  opacity  #
#############
inactive-opacity = 0.85;
active-opacity = 0.98;
frame-opacity = 1;
# inactive-opacity-override = false;
#inactive-dim = 0.2;
# inactive-dim-fixed = true;
blur-background = true;
blur-background-frame = true;
blur-background-fixed = true;
blur-kern = "7x7box";
blur-strength = 20;
# blur-kern = "5,5,5,5,4,4,4,4,3,3,3,3,1,1,1,1";
blur-background-exclude = [
	"window_type = 'dock'",
	"window_type = 'desktop'",
	"_GTK_FRAME_EXTENTS@:c"
];
# opacity-rule = [ "80:class_g = 'URxvt'" ];

############
#  fading  #
############
fading = false;
# fade-delta = 30;
fade-in-step = 0.03;
fade-out-step = 0.03;
# no-fading-openclose = true;
# no-fading-destroyed-argb = true;
fade-exclude = [ ];

###########
#  other  #
###########
log-level = "warn";
# log-file = "/home/hung/.picom.log";
backend = "glx";
mark-wmwin-focused = true;
mark-ovredir-focused = true;
# use-ewmh-active-win = true;
detect-rounded-corners = true;
detect-client-opacity = true;
refresh-rate = 0;
vsync = true;
# sw-opti = true;
unredir-if-possible = true;
# unredir-if-possible-delay = 5000;
# unredir-if-possible-exclude = [ ];
focus-exclude = [ "class_g = 'Cairo-clock'" ];
detect-transient = true;
detect-client-leader = true;
invert-color-include = [ ];
resize-damage = 1;

########################
#  glx backend stuffs  #
########################
# glx-no-stencil = true;
# glx-no-rebind-pixmap = true;
# xrender-sync-fence = true;
use-damage = true;
```

### Steps of reproduction
Run `picom --experimental-backends --backend=glx`

**Comment by yshui** [comment]
This shouldn't have any ill effect. Is this causing any problems?

**Comment by ndgnuh** [comment]
I've notice anything unusual. I opened an issue just because it tells me that it's an error and the compositor might not work correctly.

If this isn't a problem, sorry for the false alarm.

Sent from ProtonMail mobile

-------- Original Message --------

**Comment by yshui** [comment]
No problem. This is not a very good error message anyway. I should probably make it less visible.

**Comment by yshui** [comment]
Resolved by 4247d4cf78269c143728fd7b9017d72beb4b9b9d
